### ISSUE TITLE
Please tag emojis for your stickers

### ISSUE THREAD

**Issue by simonzhexu** [WhatsApp/stickers#703]
Dear developers, please check this link for more info.

https://github.com/WhatsApp/stickers/wiki/Tag-your-stickers-with-Emojis

**Comment by ProfeLuisVazquez** [comment]
If would be great if we had the final requirements of the stickers before we tag them.

**Comment by simonzhexu** [comment]
This applies to both static stickers and animated stickers. If you have apps that support static stickers already, you can update your app to support emojis.

**Comment by Dwite** [comment]
@simonzhexu just because you mentioned animated stickers, do you think it will be possible to release them to third parties this year?

**Comment by magedalqerbi** [comment]
> https://github.com/WhatsApp/stickers/wiki/Tag-your-stickers-with-Emojis

When I tag a sticker with ❤️, WhatsApp does not recognize the sticker in the love tab but when I use ♥︎ it recognize it as a Love sticker, any information about that?

**Comment by Ferdari** [comment]
Without guidelines we cannot work, what are the supported emojis?

**Comment by MarvinJWendt** [comment]
> > [WhatsApp/stickers/wiki/Tag-your-stickers-with-Emojis](https://github.com/WhatsApp/stickers/wiki/Tag-your-stickers-with-Emojis)
> 
> When I tag a sticker with ❤️, WhatsApp does not recognize the sticker in the love tab but when I use ♥︎ it recognize it as a Love sticker, any information about that?

/cc @simonzhexu

**Comment by malqerbi** [comment]
Would you give us some attention please!

**Comment by simonzhexu** [comment]
Hi Guys, I am going to get a list of emojis that we use so that you can tag those so they show up in the tabs. Sorry for the delay.

**Comment by simonzhexu** [comment]
Please use these emojis to tag:
Love

❤, 😍, 😘, 💕, 😻, 💑, 👩‍❤‍👩, 👨‍❤‍👨, 💏, 👩‍❤‍💋‍👩, 👨‍❤‍💋‍👨, 🧡, 💛, 💚, 💙, 💜, 🖤, 💔, ❣, 💞, 💓, 💗, 💖, 💘, 💝, 💟, ♥, 💌, 💋, 👩‍❤️‍💋‍👩, 👨‍❤️‍💋‍👨, 👩‍❤️‍👨, 👩‍❤️‍👩, 👨‍❤️‍👨, 👩‍❤️‍💋‍👨, 👬, 👭, 👫, 🥰, 😚, 😙, 👄, 🌹, 😽, ❣️, ❤️

Happy

😀, 😃, 😄, 😁, 😆, 😅, 😂, 🤣, 🙂, 😛, 😝, 😜, 🤪, 🤗, 😺, 😸, 😹, ☺, 😌, 😉, 🤗, 😊,

Sad

☹, 😣, 😖, 😫, 😩, 😢, 😭, 😞, 😔, 😟, 😕, 😤, 😠, 😥, 😰, 😨, 😿, 😾, 😓, 🙍‍♂, 🙍‍♀, 💔, 🙁, 🥺, 🤕, ☔️, ⛈, 🌩, 🌧,

Angry

😯, 😦, 😧, 😮, 😲, 🙀, 😱, 🤯, 😳, ❗, ❕, 🤭, 🤬, 😡, 😠, 🙄, 👿, 😾, 😤, 💢, 👺, 🗯️, 😒, 🥵

Greet

👋

Celebrate

🎊, 🎉, 🎁, 🎈, 👯‍♂️, 👯, 👯‍♀️, 💃, 🕺, 🔥, ⭐️, ✨, 💫, 🎇, 🎆, 🍻, 🥂, 🍾, 🎂, 🍰

**Comment by malqerbi** [comment]
@simonzhexu 
Thank you sir,
I see 🤭 in Angry Category, I believe it is in Happy Category!

**Comment by simonzhexu** [comment]
Good catch, removed it for now, as we need to update our code to reflect that.

**Comment by Anston** [comment]
Whatsapp allows text for searching stickers but we cannot tag stickers with text? Not all stickers can be categorized as happy, sad, angry.

**Comment by malqerbi** [comment]
@Anston just tag it with any emoji and whatsapp text search is smart enough to get the correct result

**Comment by simonzhexu** [comment]
> Whatsapp allows text for searching stickers but we cannot tag stickers with text? Not all stickers can be categorized as happy, sad, angry.

The complexity with tagging with text is that, you gotta provide all the translations for 60 languages that WhatsApp support. It is a daunting undertaking. So we have made it easier to let you tag with emoji, it is not as accurate. But you have so much less work.

**Comment by Anston** [comment]
Why do we need to? Wouldn't it suffice to add translations as needed? It's what is being done now. If its a specific emoji it gets returned. The emoji doesn't get converted into another similar one. The same could be done for text. It would be the developers responsibility to translate as needed depending on where his app is targeted for. If only english is provided then the search results will get the sticker only with the english text tag.

**Comment by chucho108** [comment]
#703

**Comment by chucho108** [comment]
`833`

**Comment by chucho108** [comment]
1.

**Comment by MarsVard** [comment]
> > Whatsapp allows text for searching stickers but we cannot tag stickers with text? Not all stickers can be categorized as happy, sad, angry.
> 
> 
> 
> The complexity with tagging with text is that, you gotta provide all the translations for 60 languages that WhatsApp support. It is a daunting undertaking. So we have made it easier to let you tag with emoji, it is not as accurate. But you have so much less work. 

Is that really true? Users create and share stickers with their community which mostly speak the same language as them

**Comment by Aitanaasecas** [comment]
Here are the emojis I use to indicate other types of stickers ( Not suitable for minors) 

  "identifier": "notsuitableforminors",
  "name": "No Apto Para Menores",

  "stickers": [
    { "image_file": "censurado.png", "emojis": ["🔞", "🚫", "😏", "🍑", "🍆"] },
    { "image_file": "sexo.png", "emojis": ["🔥", "💋", "😈", "💦"] },

**Comment by santi-petersen** [comment]
> Here are the emojis I use to indicate other types of stickers ( Not suitable for minors)
> 
> "identifier": "notsuitableforminors", "name": "No Apto Para Menores",
> 
> "stickers": [ { "image_file": "censurado.png", "emojis": ["🔞", "🚫", "😏", "🍑", "🍆"] }, { "image_file": "sexo.png", "emojis": ["🔥", "💋", "😈", "💦"] },

You cannot exceed 3 emojis per sticker

```
static final int EMOJI_MAX_LIMIT = 3;

if (sticker.emojis.size() > EMOJI_MAX_LIMIT) {
            throw new IllegalStateException("emoji count exceed limit, sticker pack identifier: " + identifier + ", filename: " + sticker.imageFileName);
}
```

And I understand that only the preset emojis are valid... but I didn't check this. (https://github.com/WhatsApp/stickers/issues/703#issuecomment-745696947)
